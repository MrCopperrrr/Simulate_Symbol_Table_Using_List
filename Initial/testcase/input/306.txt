BEGIN
INSERT y string
PRINT
BEGIN
INSERT z number
PRINT
BEGIN
INSERT a string
PRINT
END
PRINT
END
PRINT
END
PRINT