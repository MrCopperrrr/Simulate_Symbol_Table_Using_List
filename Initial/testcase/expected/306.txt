success
y//1
success
y//1 z//2
success
y//1 z//2 a//3
y//1 z//2
y//1
